src/client/exec.ts

```diff
@@ -0,0 +1,30 @@
+import { window, Terminal } from 'vscode';
+import { getVExecCommand, getCwd } from './utils';
+import { exec } from 'child_process';
+
+let vRunTerm: Terminal = null;
+
+export function execVInTerminal(args: string) {
+	const cmd = getVExecCommand(args);
+
+	window.activeTextEditor.document.save();
+	if (!vRunTerm) vRunTerm = window.createTerminal('V');
+	vRunTerm.show();
+	vRunTerm.sendText(cmd);
+}
+
+export function execV(args: string, callback: Function) {
```

Comment:
Alex, this callback will be forwarded to `child_process.exec` callback. 
I found the type of this callback
```ts
type ExecCallback = (
	error: ExecException | null,
	stdout: string | Buffer,
	stderr: string | Buffer
) => void;
```

So it would be better if you add a new definition file that might contain something like this.
```ts
import { ExecException } from 'child_process';

type ExecCallback = (
	error: ExecException | null,
	stdout: string | Buffer,
	stderr: string | Buffer
) => void;
```

Reply:
```suggestion
export function execV(args: string, callback: ExecCallback) {
```